<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>questionerFilterDialog</class>
 <widget class="QDialog" name="questionerFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>查询</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>40</y>
     <width>181</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="gradeLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="questionCntLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="gradeLabel">
      <property name="text">
       <string>等级</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="nicknameLabel">
      <property name="text">
       <string>昵称</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="questionCntLabel">
      <property name="text">
       <string>出题数</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nicknameLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
